templates/jecoute/news/_form.html.twig

```diff
@@ -22,13 +22,15 @@
     {{ form_errors(form.zone) }}
 </div>
 
+{% if isNotifiable %}
```

Comment:
1/ twig var names should be in snake case => `is_notifiable` (not mandatory)

2/ can't you retrieve it from the form options ? 
e.g. : add the option in the `buildView` method of the form type, then retrieve it with `form.vars.is_notifiable`